docs/topic-guides/system-concepts/resource-pool.txt

```diff
@@ -34,17 +34,17 @@ cluster on AWS or GCP via ``det deploy``.
 Here are some scenarios where it can be valuable to use multiple
 resource pools:
 
--  *Use GPU for training while using CPUs for TensorBoard.*
+-  *Use GPU for training while using CPUs for Tensorboard.*
```

Comment:
"TensorBoard" is correct. If this is inconsistent with something else, that should be fixed instead.

Reply:
All tensorboard changes were removed.